corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views/query.js

```diff
@@ -204,10 +204,9 @@ hqDefine("cloudcare/js/formplayer/menus/views/query", function () {
         },
 
         _isValid: function () {
-            if (!this.model.get('required')) {
+            if (!this.model.get('required') && !this.model.get("error")) {
```

Comment:
will this just trigger for required questions (hard to tell just from this snippet)? And if so would we want to check if any questions (even if it's not required) have an error?

Reply:
Good catch, looking at this again it does need an additional check for an error or a separate check where it returns false if there is an error. Might have been obvious if I were testing the entire feature instead of mocking errors.